installers/windows/minikube.nsi

```diff
@@ -0,0 +1,131 @@
+# This installs two files, minikube.exe and logo.ico, creates a start menu shortcut, builds an uninstaller, and
```

Comment:
A good example of a nice MUI installer is notepad++.  It isn't to much to add you just need a few images and swap out your pages with the MUI ones.